taxonomies/categories.txt

```diff
@@ -7653,11 +7653,12 @@ nl:Oplosdranken, Warme oplosdranken, Warme oplosdrank
 ru:Быстрорастворимые горячие напитки
 
 <en:Plant-based foods and beverages
-en:Dairies substitutes
+en:Dairy substitutes
```

Comment:
When you rename the main entry, can you add it as a synonym? Otherwise the old products will have both the new entry (taxonomized) and old entry (not taxonomized)

Reply:
OK

Reply:
it's strange, I see you commit that added the synonym, but I don't see it in the resulting changes